tests/foreman/ui/test_ansible.py

```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
             ansible_roles_table = session.host_new.get_ansible_roles(target_sat.hostname)
             assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list('8')
```

Comment:
```suggestion
    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

Reply:
Added the default RHEL version and removed hardcoded values.